zimbra/src/main/java/fr/openent/zimbra/service/impl/ReturnedMailService.java

```diff
@@ -246,13 +246,16 @@ private void recursiveDeleteMailLoop(JsonArray returnedMails, JsonArray returned
             } else {
                 this.recursiveDeleteMailLoop(returnedMails, returnedMailsStatut, index + 1, result);
             }
-            ;
         });
     }
 
     private void addDeleteFutures(JsonObject returnedMail, Handler<Either<String, JsonObject>> result) {
         JsonArray userIds = new JsonArray(returnedMail.getString(NOTIF_RECIPIENT));
         int j = 0;
+        if (userIds.size() == 0) {
```

Comment:
Vérifie si c'est null et puis tu as la méthode `isEmpty`

```suggestion
        if (userIds == null || userIds.isEmpty()) {
```